tests/foreman/longrun/test_inc_updates.py

```diff
@@ -16,279 +16,240 @@
-
-        # Promote content view to 'DEV' and 'QE'
-        assert len(self.rhel_6_partial_cv.version) == 1
-        for env in (self.dev_lce, self.qe_lce):
-            promote(self.rhel_6_partial_cv.version[0], env.id)
-
-        # Create host collection
-        self.rhel_6_partial_hc = entities.HostCollection(
-            organization=self.org, name=gen_alpha(), max_hosts=5
-        ).create()
-
-        # Create activation key for content view
-        kwargs = {'organization': self.org, 'environment': self.qe_lce.id}
-        rhel_6_partial_ak = entities.ActivationKey(
-            name=gen_alpha(),
-            content_view=self.rhel_6_partial_cv,
-            host_collection=[self.rhel_6_partial_hc],
-            **kwargs,
-        ).create()
-
-        # Fetch available subscriptions
-        subs = entities.Subscription(organization=self.org).search()
-        assert len(subs) > 0
-
-        # Add default subscription to activation key
-        sub_found = False
-        for sub in subs:
-            if sub.name == DEFAULT_SUBSCRIPTION_NAME:
-                rhel_6_partial_ak.add_subscriptions(data={'subscription_id': sub.id})
-                sub_found = True
-        assert sub_found
-
-        # Enable product content in activation key
-        rhel_6_partial_ak.content_override(
-            data={'content_overrides': [{'content_label': REPOS['rhst6']['id'], 'value': '1'}]}
-        )
-
-        # Create client machine and register it to satellite with
-        # rhel_6_partial_ak
-        self.vm = VirtualMachine(distro=DISTRO_RHEL6, tag='incupdate')
-        self.addCleanup(vm_cleanup, self.vm)
-        self.setup_vm(self.vm, rhel_6_partial_ak.name, self.org.label)
-        self.vm.enable_repo(REPOS['rhva6']['id'])
-        timestamp = datetime.utcnow()
-        self.vm.run(f'yum install -y {REAL_0_RH_PACKAGE}')
-
-        # Find the content host and ensure that tasks started by package
-        # installation has finished
-        host = entities.Host().search(query={'search': f'name={self.vm.hostname}'})
-        wait_for_tasks(
-            search_query='label = Actions::Katello::Host::UploadPackageProfile'
-            ' and resource_id = {}'
-            ' and started_at >= "{}"'.format(host[0].id, timestamp)
-        )
+from robottelo.constants.repos import FAKE_9_YUM_REPO
+from robottelo.hosts import ContentHost
+
+pytestmark = [pytest.mark.run_in_one_thread]
+
+
+@pytest.fixture(scope='module')
+def module_lce_library(module_manifest_org):
+    """ Returns the Library lifecycle environment from chosen organization """
+    return (
+        entities.LifecycleEnvironment()
+        .search(
+            query={'search': f'name={ENVIRONMENT} and organization_id={module_manifest_org.id}'}
+        )[0]
+        .read()
+    )
+
+
+@pytest.fixture(scope='module')
+def dev_lce(module_manifest_org):
+    dev_lce = entities.LifecycleEnvironment(name='DEV', organization=module_manifest_org).create()
```

Comment:
Consider just one-lining methods like these to directly return the instance that's created. Totally optional, just no use in creating a variable name that's only used once in the scope.

Reply:
ACK

Reply:
Done